Manifest.toml

```diff
@@ -153,7 +153,7 @@ version = "0.20.0"
 
```

Comment:
I'm getting the following:
```
(PowerSimulations) pkg> resolve
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package DiffResults [163ba53b]:
 DiffResults [163ba53b] log:
 ├─possible versions are: 0.0.1-0.0.4 or uninstalled
 ├─restricted to versions 0.0.3 by an explicit requirement, leaving only versions 0.0.3
 └─restricted by julia compatibility requirements to versions: 0.0.4 or uninstalled — no versions left
```

Reply:
@claytonpbarrows try now, I just updated the Manifest.TOML